core/swift3Action/spm-build/Whisk.swift

```diff
@@ -124,31 +129,31 @@ class Whisk {
                 callback(["error": "Did not receive a response."])
             }
         }
-
+        
         do {
-            #if os(OSX)
-                let jsonData = try NSJSONSerialization.data(withJSONObject: params as! [String:AnyObject], options: [])
-            #elseif os(Linux)
-                let jsonData = try NSJSONSerialization.data(withJSONObject: params.bridge(), options: [])
-            #endif
-
+            //#if os(OSX)
+                let jsonData = try JSONSerialization.data(withJSONObject: params as! [String:AnyObject], options: [])
+            //#elseif os(Linux)
+            //    let jsonData = try JSONSerialization.data(withJSONObject: params, options: [])
+            //#endif
+            
             print(jsonData)
-
+            
             request.write(from: jsonData)
             request.end()
         } catch {
             callback(["error": "Could not parse parameters."])
         }
     }
-
+    
```

Comment:
Trailing white space everywhere 🙀 (not enforced one `*.swift` files by `scanCode.py`, so no reason to fix now).